examples/nlp/gpt/conf/gpt_sft.yaml

```diff
@@ -53,6 +53,17 @@ exp_manager:
     save_best_model: False   # need to keep this false otherwise it will create multiple last.ckpt files because restore reset the previous best model
 
 model:
+  steerlm:
+  steerable_attributes :       
```

Comment:
an indentation is missing?

Reply:
removing steerlm>steerable_attributes instead using steerable_attributes directly  in the new commit. so fixed this issue